notices/ltc-asset-transfer.vbs

```diff
@@ -56,40 +57,55 @@ changelog_display
-  Text 30, 10, 45, 10, "Case number:"
-  EditBox 80, 5, 50, 15, MAXIS_case_number
-  Text 165, 10, 60, 10, "ER date (MM/YY):"
-  EditBox 230, 5, 20, 15, renewal_footer_month
-  EditBox 255, 5, 20, 15, renewal_footer_year
-  Text 10, 35, 70, 10, "Resident First Name:"
-  EditBox 80, 30, 70, 15, client
-  Text 165, 35, 65, 10, "Spouse First Name:"
-  EditBox 230, 30, 70, 15, spouse
-  Text 15, 60, 60, 10, "Worker Signature:"
-  EditBox 80, 55, 220, 15, worker_signature
-  CheckBox 65, 80, 125, 15, "Check here to CASE:NOTE actions.", case_note_checkbox
-  ButtonGroup ButtonPressed
-    OkButton 195, 80, 50, 15
-    CancelButton 250, 80, 50, 15
-EndDialog
+
+'Listing button numbers for links
+onesource_URL = 101
+epm_URL = 102
 
 Do
     Do
-        err_msg = ""
-        Dialog Dialog1
-        cancel_without_confirmation
-        IF IsNumeric(maxis_case_number) = false or len(maxis_case_number) > 8 then err_msg = err_msg & vbNewLine & "* Enter a valid case number."
-        If trim(renewal_footer_month) = "" or len(renewal_footer_month) <> 2 then err_msg = err_msg & vbNewLine & "* Enter a two-digit renewal month."
-        If trim(renewal_footer_year) = "" or len(renewal_footer_year) <> 2 then err_msg = err_msg & vbNewLine & "* Enter a two-digit renewal year."
+      err_msg = ""
+
+      Dialog1 = ""
+      BeginDialog Dialog1, 0, 0, 301, 115, "LTC asset transfer dialog"
+        EditBox 75, 5, 50, 15, MAXIS_case_number
+        EditBox 225, 5, 20, 15, renewal_footer_month
+        EditBox 250, 5, 20, 15, renewal_footer_year
+        EditBox 75, 30, 70, 15, client
+        EditBox 225, 30, 70, 15, spouse
+        EditBox 75, 55, 220, 15, worker_signature
+        ButtonGroup ButtonPressed
+          OkButton 190, 95, 50, 15
+          CancelButton 245, 95, 50, 15
+          PushButton 20, 90, 40, 15, "OneSource", onesource_URL
+          PushButton 70, 90, 40, 15, "EPM", epm_URL
+        Text 5, 10, 45, 10, "Case number:"
+        Text 155, 10, 60, 10, "ER date (MM/YY):"
+        Text 5, 35, 70, 10, "Resident First Name:"
+        Text 155, 35, 65, 10, "Spouse First Name:"
+        Text 5, 60, 60, 10, "Worker Signature:"
+        GroupBox 15, 80, 100, 30, "LTC Asset Transfer Policies"
+      EndDialog
+
+      Dialog Dialog1
+      cancel_without_confirmation
+
+      If ButtonPressed > 100 Then
+        err_msg = "Loop"
+        If ButtonPressed = onesource_URL Then run "C:\Program Files (x86)\Microsoft\Edge\Application\msedge.exe https://www.dhs.state.mn.us/main/idcplg?IdcService=GET_DYNAMIC_CONVERSION&RevisionSelectionMethod=LatestReleased&dDocName=ONESOURCE-170126"
+        If ButtonPressed = epm_URL Then run "C:\Program Files (x86)\Microsoft\Edge\Application\msedge.exe http://hcopub.dhs.state.mn.us/epm/2_4_2_1_1.htm"
+      Else  
+        Call validate_MAXIS_case_number(err_msg, "*")
+        Call validate_footer_month_entry(renewal_footer_month, renewal_footer_year, err_msg, "*")
```

Comment:
The usage on this script is low enough where I don't think we need to auto-fill names, etc. BUT what if there was a 'fill' button next to HC renewal date that when pressed the script would go to STAT/REVW and auto-filled the Health Care ER date?